Author: Joey Hess <joeyh@debian.org>
Debian-Bug: http://bugs.debian.org/753551
Description:
 xmonad floats mplayer by default. However, Debian has switched to
 mplayer2, and so on upgrade, it will stop floating. This can be easily
 fixed in the user's config file, but here is a patch that avoids
 bothering the user with breakage on upgrade.

Index: xmonad-0.11/XMonad/Config.hs
===================================================================
--- xmonad-0.11.orig/XMonad/Config.hs	2013-01-01 02:31:47.000000000 +0100
+++ xmonad-0.11/XMonad/Config.hs	2014-07-03 07:40:08.951940477 +0200
@@ -90,6 +90,7 @@
 manageHook :: ManageHook
 manageHook = composeAll
                 [ className =? "MPlayer"        --> doFloat
+                , className =? "mplayer2"       --> doFloat
                 , className =? "Gimp"           --> doFloat ]

 ------------------------------------------------------------------------
@@ -305,4 +306,4 @@
     "-- Mouse bindings: default actions bound to mouse events",
     "mod-button1  Set the window to floating mode and move by dragging",
     "mod-button2  Raise the window to the top of the stack",
-    "mod-button3  Set the window to floating mode and resize by dragging"]
\ No newline at end of file
+    "mod-button3  Set the window to floating mode and resize by dragging"]
